jira.go

```diff
@@ -382,6 +382,40 @@ func (t *BasicAuthTransport) transport() http.RoundTripper {
 	return http.DefaultTransport
 }
 
+// PATAuthTransport is an http.RoundTripper that authenticates all requests
+// using the Personal Access Token specified.
+type PATAuthTransport struct {
+	Token string
```

Comment:
Please add a doc comment for `Token` and perhaps a link to the JIRA docs for where to get the PAT

Reply:
Good point, done!